src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/FeatureChainingWfsTest.java

```diff
@@ -408,6 +412,63 @@ public void testDescribeFeatureTypeNoTypes() {
         assertXpathCount(0, "//xsd:element", doc);
     }
 
+    /**
+     * Tests that WFS schema is never imported in a DescribeFeatureType response.
+     *
+     * <p>
+     * <strong>Remarks:</strong> this test only targets WFS 2.0, as 1.1.0 is sufficiently covered
+     * by other tests. 
+     * </p>
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testDescribeFeatureTypeNoWfsSchemaImport() throws Exception {
+        // one feature type --> schema is included, no imports
+        Document doc = getAsDOM(
+                "wfs?service=WFS&version=2.0.0&request=DescribeFeatureType&typenames=gsml:GeologicUnit");
+        assertXpathCount(0, "//xsd:import", doc);
+        assertXpathCount(1, "//xsd:include", doc);
+        assertImportNotExists(doc, WFS.NAMESPACE);
+        assertIncludeExists(doc, GSML_SCHEMA_LOCATION_URL);
+
+        // two feature types, same schema / namespace --> schema is included once, no imports
+        doc = getAsDOM(
+                "wfs?service=WFS&version=2.0.0&request=DescribeFeatureType&typenames=gsml:GeologicUnit,gsml:MappedFeature");
+        assertXpathCount(0, "//xsd:import", doc);
+        assertXpathCount(0, "//xsd:import[@namespace='" + WFS.NAMESPACE + "']", doc);
+        assertImportNotExists(doc, WFS.NAMESPACE);
+        assertIncludeExists(doc, GSML_SCHEMA_LOCATION_URL);
+
+        // two feature types, different schemas / namespaces --> first schema is included, second is imported
+        doc = getAsDOM(
```

Comment:
Thanks for adding this tests case, for reference the result of this request will be:
```
<?xml version="1.0" encoding="UTF-8"?><xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" targetNamespace="urn:cgi:xmlns:CGI:GeoSciML:2.0">
  <xsd:import namespace="http://example.com" schemaLocation="file:/home/nuno/github/geoserver-3/src/extension/app-schema/app-schema-test/target/app-schema-mock3923701145728430417data/featureTypes/ex_FirstParentFeature/simpleContent.xsd"/>
  <xsd:include schemaLocation="http://www.geosciml.org/geosciml/2.0/xsd/geosciml.xsd"/>
</xsd:schema>
```

This looks a bit strange at a first look but indeed this is compliant with OGC WFS 2.0 standard:

```
9.3.2 Supporting multiple namespaces

An XML Schema document can only declare elements in a single namespace (see W3C XML Schema Part 2).
In the event that a DescribeFeatureType operation requests that feature types in multiple namespaces be
described, the server shall generate the complete schema for one of the requested namespaces and import
the remaining namespaces. This International Standard does not mandate which namespace's schema
should be generated and which namespaces should be imported.
```